benchmarks/Kestrel.Performance/HttpParserBenchmark.cs

```diff
@@ -51,24 +51,23 @@ private void InsertData(byte[] data)
 
         private void ParseData()
         {
-            if (!_parser.ParseRequestLine(new Adapter(this), _buffer, out var consumed, out var examined))
+            BufferReader<byte> reader = new BufferReader<byte>(_buffer);
```

Comment:
nit, var. (unless ASP.NET has other convention)

Reply:
VAR ALL THE THINGS!

Reply:
'var attemptCleanup = true'